tests/ctst/common/common.ts

```diff
@@ -237,14 +237,15 @@ Then('i {string} be able to add user metadata to object {string}',
         }
     });
 
-Then('kafka consumed messages should not take too much place on disk', { timeout: -1 },
+const kafkaCleanerTimeout = 1800000; // 30min
```

Comment:
30 minutes seems verrrry long for kafka cleaner, which is setup to run every 30 second (or maybe 1-2 minutes) in CI...

Reply:
It's already 30min today, this was the config where we got the best results

Reply:
I am trying this now https://github.com/scality/Zenko/pull/2135/commits/0125893e807c72653c7605fa4f0f67178692324f